test/testlib.py

```diff
@@ -314,6 +314,7 @@ def enter_page(self, path, host=None, reconnect=True):
         while True:
             try:
                 self.wait_present("iframe.container-frame[name='%s'][data-loaded]" % frame)
+                self.wait_not_visible(".curtains")
```

Comment:
This is only in the main frame, (see the swtich to top above) other packages are always in an iframe. So something like the docker or kube curtain (which we have tests for) would never be caught here.